org.es_de.emulationstation-de.yml

```diff
@@ -0,0 +1,82 @@
+#  SPDX-License-Identifier: MIT
+#
+#  EmulationStation Desktop Edition
+#  org.es_de.emulationstation-de.yml
+#
+#  Flatpak manifest file for use with flatpak-builder.
+#
+
+app-id: org.es_de.emulationstation-de
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+command: emulationstation
+branch: stable
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --filesystem=host:ro
+  - --filesystem=/var/lib/flatpak:ro
+  - --filesystem=home
+  - --filesystem=~/.var/app
+  - --share=network
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=PATH=/app/bin:/usr/bin:/var/lib/flatpak/exports/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/var/lib/snapd/snap/bin
+cleanup:
+  - /include
+  - /share/ffmpeg
+  - /lib/cmake
+  - /lib/debug
```

Comment:
```suggestion
```

Reply:
I could remove it, but doing that will just leave me with an empty lib/debug directory after building the package. If that's something you need retained inside the Flatpak for some reason then I have no problem doing it, so please clarify whether it's required.

Reply:
debug symbols are separated into their own flatpak.

Reply:
Thanks for clarifying, I have now removed the entry and I've just committed the change.